aws/table_aws_route53_traffic_policy_instance.go

```diff
@@ -0,0 +1,201 @@
+				ShouldIgnoreErrorFunc: isNotFoundError([]string{"NoSuchTrafficPolicyInstance"}),
+			},
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listTrafficPolicyInstances,
+		},
+		Columns: awsColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The DNS name for which Amazon Route 53 responds to queries.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "id",
+				Description: "The id that Amazon Route 53 assigned to the new traffic policy instance.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "hosted_zone_id",
+				Description: "The id of the hosted zone that Amazon Route 53 created resource record sets in.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "message",
+				Description: "If State is Failed, an explanation of the reason for the failure.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "state",
+				Description: "Current state of the instance.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "traffic_policy_id",
+				Description: "The ID of the traffic policy that Amazon Route 53 used to create resource record sets in the specified hosted zone.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "traffic_policy_type",
+				Description: "The DNS type that Amazon Route 53 assigned to all of the resource record sets that it created for this traffic policy instance.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "traffic_policy_version",
+				Description: "The version of the traffic policy that Amazon Route 53 used to create resource record sets in the specified hosted zone.",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "ttl",
+				Description: "The TTL that Amazon Route 53 assigned to all of the resource record sets that it created in the specified hosted zone.",
+				Type:        proto.ColumnType_INT,
+				Transform:   transform.FromField("TTL"),
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getRoute53TrafficPolicyInstanceTurbotAkas,
+				Transform:   transform.FromValue(),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listTrafficPolicyInstances(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	// Create session
+	svc, err := Route53Service(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("aws_route53_traffic_policy_instance.listTrafficPolicyInstances", "service_creation_error", err)
+		return nil, err
+	}
+
+	input := &route53.ListTrafficPolicyInstancesInput{
+		MaxItems: aws.String("100"),
```

Comment:
Can we add optional quals column (name, hosted_zone_id, traffic_policy_type) in this table?
In doc, I can see we can pass these column values in as input.
https://docs.aws.amazon.com/Route53/latest/APIReference/API_ListTrafficPolicyInstances.html

Reply:
The variables in the input are the marker fields which are used only for pagination purposes as the description suggests (If the value of IsTruncated in the previous response was true, you have more traffic policy instances)